### PR TITLE
Fix violations of and reenable rubocop Style/MultilineMethodCallIndentation.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: bin/cron/admin_stats [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      # TODO(asher): Add more student-teacher metrics and pipe them to the admin_stats dashboard.
+      teachers: {
+        with_students: students.join(:followers, student_user_id: :id).select(:followers__user_id).distinct.count,
+        with_active_students: thirty_day_active_students.join(:followers, student_user_id: :id).select(:followers__user_id).distinct.count,
+      },
+    }
+  )
+end
+
+main
```
Comment (aoby) [inline bin/cron/admin_stats]: Is this file being re-added intentionally? It looks like it was removed in https://github.com/code-dot-org/code-dot-org/pull/14315

File: bin/cron/push_latest_rds_backup_to_secondary_account [hunk 2]
```diff
@@ -98,8 +98,8 @@ def main
 
   # Find the latest automated backup
   sorted_snapshots = production_instance.snapshots.
-      select {|snap| snap.status == 'available' && !snap.snapshot_id.start_with?('temp-snapshot-')}.
-      sort_by(&:snapshot_create_time)
+    select {|snap| snap.status == 'available' && !snap.snapshot_id.start_with?('temp-snapshot-')}.
+    sort_by(&:snapshot_create_time)
```
Comment (aoby) [inline bin/cron/push_latest_rds_backup_to_secondary_account:102]: I'm a little confused about this rule. It seems most of the changes so far are indenting 4 spaces, but this one is going back from 4 to 2?

File: dashboard/app/models/blockly.rb [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+    # Wrapped since we store our serialized booleans as strings.
+    never_autoplay_video == 'true'
+  end
+
+  def fix_examples
+    # remove nil and empty strings from examples
+    return if examples.nil?
+    self.examples = examples.select(&:present?)
+  end
+end
```
Comment (aoby) [inline dashboard/app/models/blockly.rb]: More re-added files I'm pretty sure don't belong here...

File: pegasus/helpers/section_api_helpers.rb [hunk 4]
```diff
@@ -107,10 +107,10 @@ def self.update_if_allowed(params, dashboard_user_id)
...
     return if user_to_update.empty?
     return if Dashboard.db[:sections].
-      join(:followers, section_id: :sections__id).
-      where(sections__user_id: dashboard_user_id, sections__deleted_at: nil).
-      where(followers__student_user_id: params[:id], followers__deleted_at: nil).
-      empty?
+        join(:followers, section_id: :sections__id).
+        where(sections__user_id: dashboard_user_id, sections__deleted_at: nil).
+        where(followers__student_user_id: params[:id], followers__deleted_at: nil).
+        empty?
```
Comment (aoby) [inline pegasus/helpers/section_api_helpers.rb:113]: Here's another 4-space indent. It seems inconsistent. Is there a reason some of these are 2-space and others 4? FWIW I prefer the 2-space for consistency, but I'm happy to go with whatever the linter enforces. I just don't understand what it's enforcing right now.

Comment (ashercodeorg) [reply pegasus/helpers/section_api_helpers.rb:113]: My belief is that the linter believes in four space indents following an `if` or `unless`. Granted, I'm all for consistency also. Do we want to do two everywhere, though it makes it hard to distinguish the if condition from the block inside the if? Four for line continuation? Something else?

File: dashboard/app/models/blockly.rb (hunk 3 above)
Comment (ashercodeorg) [reply dashboard/app/models/blockly.rb]: No idea how or why these got readded. Now fixed, though rebasing staging was insufficient.

File: pegasus/helpers/section_api_helpers.rb (hunk 4 above)
Comment (aoby) [reply pegasus/helpers/section_api_helpers.rb:113]: OIC. That makes sense. This does seem consistent with the double indent under if or unless.  I didn't notice that before.

**Pull request closed without merging**